include/IMeshBuffer.h

```diff
@@ -649,6 +674,14 @@ GL_INVALID_OPERATION is generated by glVertexAttribPointer if size is GL_BGRA an
             return ((uint8_t*)mappedAttrBuf->getPointer())+ix;
         }
 
+		//! Accesses given index of given attribute. Index number is incremented by `baseVertex`.
+		/** If component count of given attribute is less than 4, only first ones of output vector's members will be written.
+		@param[out] output Vector object to which index's value will be returned.
+		@param[in] attrId Atrribute id.
+		@param[in] ix Index which is to be accessed. Will be incremented by `baseVertex`.
+		@returns true if successful or false if an error occured (e.g. no such index or given atrribute's type not implemented).
```

Comment:
ix out of range, no attribute specified/bound or given attribute's format conversion to vectorSIMDf unsupported